sdk/cosmosdb/cosmos/src/request/DedicatedGatewayRequestOptions.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Dedicated Gateway Request Options
+ */
+export interface DedicatedGatewayRequestOptions {
+  /**
+   * Sets the staleness value associated with the request in the Azure CosmosDB service. For requests where the {@link
+   * com.azure.cosmos.ConsistencyLevel} is {@link com.azure.cosmos.ConsistencyLevel#EVENTUAL}, responses from the
+   * integrated cache are guaranteed to be no staler than value indicated by this maxIntegratedCacheStaleness.
+   *
+   * <p>Default value is null</p>
```

Comment:
Maybe a note here about what the server does when this is not set? Assuming there is no limit or there is some default one?